tests/conftest.py

```diff
@@ -37,6 +36,7 @@ def remove_storage_file(request):
 def remove_storage_file_after(request):
     storage_file = get_datafile_filename(request.node)
     yield storage_file
-    PersistentObjectStorage().dump()
+    # TODO: found why this were necessary before change of singletons
+    # PersistentObjectStorage().cassette.dump()
     if storage_file.is_file():
```

Comment:
The file was not saved in that time so I force the dump to save it in this place.

Reply:
it is first issue with removing of singletons. because now there is no link to cassette, because there are used own cassette instances (in fact. if no cassette is given then it uses cassette from `PersistentObjectStorage()` but it is just for backward compatibility)

Reply:
Good point. We need to create a new local instance of cassette here probably. (With the correct name.)